lib/moon_web/pages/getting_started_page.ex

```diff
@@ -39,7 +38,7 @@ defmodule MoonWeb.Pages.GettingStartedPage do
           class="p-0.5 rounded-lg flex bg-beerus-100"
         />
       </div>
-      <ForDesigner :if={@selected_role == List.first(@roles)} socket={@socket} />
+      <ForDesigner :if={@selected_role == List.first(@roles)} />
```

Comment:
Why this does not check:

`@selected_role == "I'm a designer"`

or 

`@selected_role == @designer_role`

Too complex?

Reply:
done, but with functions - they're available boith in mount() and render()